package.json

```diff
@@ -14,8 +14,8 @@
     "simple-spinner": "0.0.5"
   },
   "devDependencies": {
-    "eslint": "3.3.1",
-    "eslint-plugin-shopify": "14.0.2"
+    "eslint": "3.7.1",
```

Comment:
we're updating the peer dependency so this will be out of date by later today.  I'll ping again once its merged so you can update here, or just merge this and we can update later